### DISCUSSION TITLE
Make the language amount line up with the language

### DISCUSSION THREAD

**Discussion by Milo123459** [lowlighter/metrics#459] [Ideas]
![image](https://user-images.githubusercontent.com/50248166/128664714-662e65c3-713d-40da-8c54-d9f0b2cbaef9.jpeg)

As you can see they don't line up with the fields and I find it a bit annoying. Might be cool if for big fields like that if the language, percentage and size would line up with the bar.

**Comment by Nixinova** [comment]
What so you mean 'line up'? Removing the tab between the language name and the stats, or aligning each language with the bar above it and having the stats below?

If the latter:
If you have say 8 languages, how would you lay it out? It doesn't seem like you could do that and make them fit. And what about languages that are only 1%?

**Reply by Milo123459** [reply]
The latter, and maybe just have a section below the bars for the ones where they can't be displayed

**Reply by lowlighter** [reply]
There are already some display issues depending on long languages names and if you're enabling multiple options (bytes size + lines + percentage), so half-width is already pretty tight to fit everything in it.

I think it would be hard to implement what's suggested. 

Multiple display styles for a legend is a bit confusing so I'd be in favor of having same display whether a language is 100% or 1%
